modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -68,18 +68,18 @@ func (k Keeper) GetChecksums() collections.KeySet[[]byte] {
 	return k.checksums
 }
 
-// GetQueryPlugins returns the set query plugins.
-func (k Keeper) GetQueryPlugins() QueryPlugins {
```

Comment:
This is API breaking for 08-wasm, right? I added this PR to the [list of PRs to go through when updating the docs and migration docs](https://github.com/cosmos/ibc-go/issues/6139), but should we at least reflect that is API breaking in the title of the PR?